frontend/packages/process-editor/src/components/ConfigPanel/ConfigPanel.tsx

```diff
@@ -42,7 +42,8 @@ const ConfigPanelContent = (): React.ReactElement => {
     return <ConfigSequenceFlow key={bpmnDetails.id} />;
   }
 
-  const isSupportedConfig = bpmnDetails.type === BpmnTypeEnum.Task;
+  const isSupportedConfig =
+    bpmnDetails.type === BpmnTypeEnum.Task || bpmnDetails.type === BpmnTypeEnum.EndEvent;
```

Comment:
Is this change suppose to be here? 🤔

Reply:
I think so? As the EndEvent is also supported config 😄